Cargo.lock

```diff
@@ -4869,6 +4892,17 @@ dependencies = [
  "winapi 0.3.9",
 ]
 
+[[package]]
+name = "stacks-codec"
+version = "2.4.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "9d5791855bf9de767f55042f1a034a2575373c6433b0965f1efac14e7215f71b"
+dependencies = [
+ "clarity-vm",
+ "serde",
+ "wsts 8.1.0",
+]
+
 [[package]]
 name = "stacks-codec"
```

Comment:
We shouldn't have 2 versions of stacks-codec. Especially since [this PR](https://github.com/hirosystems/chainhook/pull/617) was merged

Running `cargo check` (or `cargo update chainhook-sdk`)  should fix it